### DISCUSSION TITLE
A collection of dictionaries for auto-complete lovers

### DISCUSSION THREAD

**Discussion by liuh886** [tadashi-aikawa/obsidian-various-complements-plugin#297] [Show and tell]
I have to say, the auto-complete feature is absolutely amazing! 

In order to make sure I had the best dictionary possible, I spent a whole weekend extracting my own phrases from various PDFs and documents. Initially, my goal was to use this for academic writing, so I focused on extracting phrases from Elsevier corpus. Now,  I turn it into a side project related to Various Complements:

[**Open Phrasebank**: Help users build customized phrasebanks from various texts or corpora.](https://github.com/liuh886/open_phrasebank)

I have created dictionaries for `Environment & Earth Science`, `Social Science`, and `Medicine`, I am happy to share them with others. 

If you have a favorite dictionary of your own, I would love for you to share it with me!

![image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/7567311/63c72b63-2cd5-49a3-a570-1333ea395b12)

**Comment by tadashi-aikawa** [comment]
Thank you for sharing, @liuh886 :)

I do not include a dictionary for general terms, but this approach will be helpful to many people. I will introduce it in the Various Complements documentation soon :+1:

**Reply by tadashi-aikawa** [reply]
@liuh886 
Added to https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/6.%20Showcase/%F0%9F%93%93%20Custom%20dictionary%20examples/#open-phrasebank 👍 

![image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/9500018/c565c0b8-85d1-4d60-a87c-e19f7b692441)
